### PR TITLE
README: general project description

### PR DESCRIPTION
**Pull request by mareklibra**
Created together: [Milestones](https://github.com/oVirt/ovirt-web-ui/milestones)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-POC of new oVirt User Portal.
+# User Portal for oVirt
+Fancy looking UI for standard (non-admin) oVirt users.
+
+This project is meant to be full replacement for existing oVirt Basic User Portal.
```
Comment (jelkosz) [inline README.md]: I would rephrase it to something like:
"This project aims to be a drop-in replacement of the existing Basic User Portal which includes selected functionality of the Power User Portal."

File: README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+Revised list of Extended User Portal features will be implemented to ideally replace it as well.
+
+Performance bottlenecks of previous Basic User Portal are addressed.
```
Comment (jelkosz) [inline README.md]: Also just rephrase to:
"The application is built with performance in mind".

File: README.md [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+
+This project is not intended to be full-feature oVirt admin UI, it's focus is on standard user with no or limited administration skills or privileges.
+
+For list of features please refer the [Milestones](https://github.com/oVirt/ovirt-web-ui/milestones) or [Issues](https://github.com/oVirt/ovirt-web-ui/issues).
```
Comment (jelkosz) [inline README.md]: I would maybe add some sentence like:
## Fist Milestone
The first milestone should support a subset of the functionality of the current Basic User Portal, namely:
- list of VMs (not pools)
- basic operations (start/stop/...)
- take console
- present VM details
For details: <link to the milestone>

## Second milestone
Should include all features of the Basic User Portal and include selected functionality of the Power User Portal, namely:
- add/edit VM (basic properties only)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
-POC of new oVirt User Portal.
+# User Portal for oVirt
+Fancy looking UI for standard (non-admin) oVirt users.
```
Comment (gregsheremeta) [inline README.md]: Suggest "NG" since we use that a lot in oVirt.

-1 to "Fancy looking" for sure.
